src/App.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+
+import {
+  Redirect,
+  Route,
+  BrowserRouter as Router,
+  Switch
+} from 'react-router-dom';
+
+import config from './config';
+
+import Home from './pages/Home';
+import About from './pages/About';
+import Analysis from './pages/Analysis';
+import Contact from './pages/Contact';
+import Legal from './pages/Legal';
+import NotFound from './pages/404';
+import Search from './pages/Search';
+import ScrollToTop from './components/ScrollToTop';
+
+const supportedCountries = config.countries.reduce(
+  (previousValue, currentValue) =>
+    previousValue ? `${previousValue}|${currentValue.slug}` : currentValue.slug,
+  undefined
+);
+
+/**
+ * NOTE: We use redirect to ensure all urls end with `/` consistently.
+ *       Otherwise `/about` and `/about/` would be treated as 2 URLs for SEO, etc.
+ */
+function App() {
+  return (
+    <Router basename={process.env.PUBLIC_URL}>
+      <ScrollToTop>
```

Comment:
This is going to be annoying if I want a page to load and scroll to position. 
Maybe it should be in the Page component and has a prop true|false or take id of element to scroll to.

Reply:
Since this happens at router level @karimkawambwa, it has no impact on the page when rendered... `/about`, `/terms`, etc., all have scroll within their respective pages and still works as expected.